activerecord/test/schema/schema.rb

```diff
@@ -1411,6 +1411,10 @@
     t.bigint :toooooooo_long_a_id, null: false
     t.bigint :toooooooo_long_b_id, null: false
   end
+
+  create_table :pk_with_defaults, id: false, force: true do |t|
+    t.bigint :id, default: 123
```

Comment:
One issue I see is that this isn't a primary key (which means there's no uniqueness constraint)